src/Reveal.Sdk.Dom.Tests/Visualizations/CandleStickVisualizationFixture.cs

```diff
@@ -0,0 +1,362 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class CandleStickVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var candleStickVisualization = new CandleStickVisualization();
+
+        // Assert
+        Assert.NotNull(candleStickVisualization);
+        Assert.Equal(ChartType.Candlestick, candleStickVisualization.ChartType);
+        Assert.Null(candleStickVisualization.Title);
+        Assert.Null(candleStickVisualization.DataDefinition);
+        Assert.NotNull(candleStickVisualization.Closes);
+        Assert.Empty(candleStickVisualization.Closes);
+        Assert.NotNull(candleStickVisualization.Highs);
+        Assert.Empty(candleStickVisualization.Highs);
+        Assert.NotNull(candleStickVisualization.Lows);
+        Assert.Empty(candleStickVisualization.Lows);
+        Assert.NotNull(candleStickVisualization.Opens);
+        Assert.Empty(candleStickVisualization.Opens);
+        Assert.Equal(0, candleStickVisualization.ColumnSpan);
+        Assert.Equal(0, candleStickVisualization.RowSpan);
+        Assert.Null(candleStickVisualization.Description);
+        Assert.NotNull(candleStickVisualization.Labels);
+        Assert.Empty(candleStickVisualization.Labels);
+        Assert.True(candleStickVisualization.IsTitleVisible);
+    }
+
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void Constructor_InitializesCandleStickVisualizationWithDataSourceItem_WhenDataSourceItemIsProvided(
+        bool hasTabularData)
+    {
+        // Arrange
+        var dataSourceItem = new DataSourceItem { HasTabularData = hasTabularData };
+
+        // Act
+        var candleStickVisualization = new CandleStickVisualization(dataSourceItem);
+
+        // Assert
+        Assert.NotNull(candleStickVisualization);
```

Comment:
Remove this. As the codes below already ensure that candleStickVisualization is not null